package.json

```diff
@@ -32,7 +33,6 @@
   },
   "scripts": {
     "check": "tsc --noEmit",
-    "export": "npm run check && ./export.sh",
```

Comment:
Why did we remove `export`? There is no other command that allows us to "rebuild" `npm` and `jsr` right?

Reply:
Since the type check is now rolled up in the tests, this seemed redundant (we just need to run `./export.sh`). But i think you're right, I'll put it back in. I expect this will change a bit anyway when we reorganise the repo and automate publish.